docs/documentation/release_notes/topics/22_0_0.adoc

```diff
@@ -88,4 +88,10 @@ In relation to the KeyStore Vault news, we also integrated Quarkus's recently re
 
 = Account Console v3 promoted to preview
 
-In version 21.1.0 of Keycloak the new Account Console (version 3) was introduced as an experimental feature. Starting this version it has been promoted to a preview feature.
\ No newline at end of file
+In version 21.1.0 of Keycloak the new Account Console (version 3) was introduced as an experimental feature. Starting this version it has been promoted to a preview feature.
+
+= Account Console Feature Flags Removed
+
+Two of the feature flags available to Account Console V2 and V3 were unused.  They were therefore removed.  It is possible that if a developer extended the account console, he or she could make use of thse flags.  However, it would not be recommended.
+
+The flags are `isEventsEnabled` and `isTotpConfigured`.
```

Comment:
I think calling these feature flags makes it difficult to understand. They are only exposed as variables in the FreeMarker template:

```suggestion
= Account Console template variables removed

Two of the variables exposed to the Account Console V2 and V3 templates (`isEventsEnabled` and `isTotpConfigured`) were left unused, and have been removed in this release.

It is possible that if a developer extended the Account Console theme, he or she could make use of these variables. So make sure that these variables are no longer used if you are extending the base theme.
```